tests/sanity_test.py

```diff
@@ -7,7 +7,7 @@ def test_smart_canvas(self):
         from smart_canvas.core import CanvasCore
         from queue import Queue
         q_producer = Queue(maxsize=1)
-        core = CanvasCore(q_producer)
+        core = CanvasCore(q_producer, screensize=(1280,720))
```

Comment:
There is a reference in web-service too [/web/main/events.py#L28](https://github.com/CumbersomePack/SmartCanvas/blob/ISSUE-59-SO/web/main/events.py#L28), so add it there too, otherwise the web-service breaks.
There is no test for the socketio stuff yet so I just have to point this out manually.